### DISCUSSION TITLE
Replace Snap with Flatpak now that Electron supports it

### DISCUSSION THREAD

**Discussion by SpaceRaccoonFluff** [standardnotes/forum#2559] [Suggestions]
Basically seeking an update to https://github.com/standardnotes/desktop/issues/172#issuecomment-820655496 

None of us linux users like snap when flatpak respects users far better. An opensource private note taking app shouldn't be supporting a closed source back-end snap either.  

From what I'm seeing electron now makes it just as easy to publish a flatpak as they do with snaps https://github.com/electron-userland/electron-builder/pull/5711 and i'm wondering if there's been any updates or plans to make the switch since then? All the linux users would love to use flatpak over snaps. I personally wouldn't even mind seeing support for app images go away if that means you guys can just focus on only supporting flatpaks. Maintaining both app images and snaps are redundant but many wouldn't give up app images if snaps are the only alternative.

**Comment by adrihut** [comment]
Currently looking into this, @SpaceRaccoonFluff. Will post updates here as soon as they become available.

**Comment by stkr22** [comment]
Any chance this is supported officially soon? There is a flatpak(https://github.com/flathub/org.standardnotes.standardnotes) but it is not official and I would rather use an supported one (even if that one looks ok).

**Comment by effieeee** [comment]
No plans on this just yet 😞 Though yes, I haven't yet heard much complaints about the 3rd party flatpak that was made

**Comment by SpaceRaccoonFluff** [comment]
Snap version broke again while updating to Fedora 38. Is there a reason flatpack is not officially supported yet? It makes a lot of sense towards the linux community. There is already a .deb download available making a snap redundant while a flatpak would help out all the linux distributions that aren't Ubuntu. It's the sandboxing of the flatpack over app-images, less errors, and compatibility with many more linux distributions that I appreciate about flatpacks.

**Comment by SpaceRaccoonFluff** [comment]
Not good, snap support compared to flatpak is dangerous: https://github.com/nextcloud-snap/nextcloud-snap/wiki/Why-Ubuntu-is-the-only-supported-distro

**Comment by 12people** [comment]
@haflaxa Any updates on this? Wouldn't it suffice to simply take over the support for the [unofficial Flatpak](https://flathub.org/apps/org.standardnotes.standardnotes)?

It's a bit strange that even though Standard Notes claims to value open-source and security, the only Linux distribution options are the proprietary-backend Snap and the relatively insecure (no sandboxing by default) AppImage.

**Comment by glidingthrough** [comment]
Another longtime user here who would appreciate a verified and officially maintained Flatpak version. Updates on this front would be welcome 🙏 Thank you!

**Comment by R25114B** [comment]
Please release an official flatpak version. AppImages are awkward to update.
